airmozilla/base/templates/500.html

```diff
@@ -4,16 +4,29 @@
     <meta content="text/html; charset=UTF-8" http-equiv="Content-Type">
     <title>Something went wrong!</title>
     <style type="text/css">
-      body {
-        font-family: "Open Sans Light","Lucida Sans","Lucida Grande","Lucida Sans Unicode",Verdana,sans-serif;
-        background-color: rgb(32, 39, 42);
-        color: rgb(255, 255, 255);
-      }
+      body {font-family: "Open Sans Light","Lucida Sans","Lucida Grande","Lucida Sans Unicode",Verdana,sans-serif;
+        background-color:rgb(32, 39, 42);}
+      h1 {color:white}
+      h2 {color:white}
+      h3 {color:red}
+      p {color:white}
+      h4 {position:fixed;
+          width:100%;
+          top:90%;
+          text-align: center;}
+      a {color:#73BAE8;}
     </style>
   </head>
   <body>
-    <h1>Server Error</h1>
-    <p>Sorry, Air Mozilla is currently experiencing an error.</p>
-    <p>We've been notified about the problem and we'll fix it soon!</p>
+    <div align="center">
+      <h1><a href="https://air.mozilla.org/"><img src="https://air.cdn.mozilla.net/static/img/logo-airmo-md.png" align = "middle"></a></h1>
+        <h2>Server Error</h2>
+        <h3>OOPS!! &nbsp; Error "500"</h3>
+        <p>Sorry, Air Mozilla is currently experiencing an error.</p>
+        <p>We've been notified about the problem and we'll fix it soon!</p>
+      </div>
+      <h4><a href="https://bugzilla.mozilla.org/enter_bug.cgi?format=guided#h=dupes%7CAir+Mozilla%7COther">
+            Report the issue at Bugzilla</a>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
+            &nbsp;&nbsp;&nbsp;<a href="https://search.mibbit.com/search/airmozilla">Connect to airmozilla IRC</a></h4> 
```

Comment:
I have no idea what `https://search.mibbit.com/search/airmozilla` is but it's certainly not a way to connect to IRC :)